### PR TITLE
PageHeader to Upload CV, issue_41 done

### PR DESCRIPTION
**Pull request by CharlesKimMaina**
# Description

A search page header is created where you can upload your CV,.,(Nb, currently the button is STATIC, it will fetch CVs later in the project)

Fixes # 41

# How to test?
1. run npm run story and see the component come up as below.
Header4UploadingCV_storybook

2. You can also test it in localhost npm run dev but u must import PageHeader component in app.js (Client-side), and the component will be like below; both at full screen and at mobile platform;;;
Header4UploadingCV
Header4UploadingCV_mobile

# Checklist

- [*] I have performed a self-review of my own code
- [*] I have commented my code, particularly in hard-to-understand areas
- [*] I have made corresponding changes to the documentation
- [*] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by kamalrsa** [comment]
looks nice !!

File: src/client/components/PageHeaderComponent/PageHeader.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+import Search from '../SearchComponent/Search';
+import Button from '../Button/Button';
+import './PageHeader.styles.css';
+
+export default function PageHeader() {
+  return (
+    <div>
+      <div className="cv-loader">
+        <div>Uploaded Resume</div>        
+          <div className = "cv-btn">
```
Comment (benna100) [inline src/client/components/PageHeaderComponent/PageHeader.js:11]:
```suggestion
          <div className = "cv-button">
```

File: src/client/components/PageHeaderComponent/PageHeader.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            <Button buttonName={'Upload New Cv'} backgroundColor = {("backgroundColor", "black")} />
```
Comment (benna100) [inline src/client/components/PageHeaderComponent/PageHeader.js:12]: Fix the error here!

File: src/client/components/PageHeaderComponent/PageHeader.styles.css [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  color: white;
+  border-radius: 6px;
+}
+.search-button {
+  margin-left: 130px;
+  margin-top: -8px;
+  margin-right: 130px;  
+}
+
+@media(max-width: 450px){
```
Comment (benna100) [inline src/client/components/PageHeaderComponent/PageHeader.styles.css:20]: Great with the responsive design

**Review by benna100 — CHANGES_REQUESTED** [review]
Small changes, otherwise looks good!

File: src/client/components/PageHeaderComponent/PageHeader.js [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  return (
+    <div>
+      <div className="cv-loader">
+        <div>Uploaded Resume</div>        
+          <div className = "cv-button">
+            <Button buttonName="Upload New Cv" style = {{ backgroundColor: 'black', color: "white" }}/>
+          </div>
+        </div>
+      <div className="search-button">
+        <Search />
```
Comment (benna100) [inline src/client/components/PageHeaderComponent/PageHeader.js:16]: Indentation. No need to fix it. BUT this should not be something i should need to comment at this stage! Use prettier that is what it is there for. Not indenting your code is a big red flag in my opinion. So remember that!

File: src/client/components/PageHeaderComponent/PageHeader.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+import Button from '../Button/Button';
+import './PageHeader.styles.css';
+
+export default function PageHeader() {
+  return (
+    <div>
+      <div className="cv-loader">
+        <div>Uploaded Resume</div>        
+          <div className = "cv-button">
+            <Button buttonName="Upload New Cv" style = {{ backgroundColor: 'black', color: "white" }}/>
```
Comment (benna100) [inline src/client/components/PageHeaderComponent/PageHeader.js:12]:
```suggestion
            <Button buttonName="Upload New Cv" style={{ backgroundColor: 'black', color: "white" }}/>
```

File: src/client/components/PageHeaderComponent/PageHeader.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+          </div>
+        </div>
+      <div className="search-button">
```
Comment (benna100) [inline src/client/components/PageHeaderComponent/PageHeader.js:15]: This is but a button but a div. I will accept it. But remember these things

**Review by benna100 — APPROVED** [review]
Looks good! Remember indentation. Really remember it. Install prettier so that it helps you and you dont even need to think about it!

**Pull request merged**